vortex_msgs/msg/CameraFeedSelection.msg

```diff
@@ -0,0 +1,3 @@
+# Message for selecting active cameras in each feed
```

Comment:
Be more specific about how the fields should be set. Does this work so that you send one message each time you want to switch camera for one of the feeds?